src/styles.css

```
@@ -209,6 +325,51 @@ body.modal-open{overflow:hidden;  height:100vh;}
+.org-card.focused{outline:2px solid rgba(255,122,0,0.12);outline-offset:2px}
+
+/* Primary buttons */
+#surpriseBtn, .btn-primary{background:linear-gradient(90deg,var(--orange),var(--orange-deep));color:var(--white);box-shadow:0 10px 30px var(--accent-glow);border:none;border-radius:14px;padding:12px 18px;transition:transform .18s,box-shadow .18s}
+#surpriseBtn:hover, .btn-primary:hover{transform:translateY(-2px);box-shadow:0 18px 40px var(--accent-glow)}
+
+/* Timeline active node glow */
+#timeline .pulse-dot{box-shadow:0 0 0 8px rgba(255,122,0,0.08),0 0 20px rgba(255,122,0,0.28)}
+
+/* Focus visibility for keyboard users */
+:focus{outline:none}
+:focus-visible{outline:3px solid rgba(255,122,0,0.12);outline-offset:2px}
```

Stylelint (17.12.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)